README.md

```diff
@@ -32,610 +34,107 @@ For more information refer to [IntelliJ IDEA documentation](https://www.jetbrain
-The recording will contain the following event types:
- - `jdk.ExecutionSample`
- - `jdk.ObjectAllocationInNewTLAB` (alloc)
- - `jdk.ObjectAllocationOutsideTLAB` (alloc)
- - `jdk.JavaMonitorEnter` (lock)
- - `jdk.ThreadPark` (lock)
-
-To start profiling cpu + allocations + locks together, specify
+In a typical use case, profiling a Java application is just a matter of a running `asprof` with a PID of a
+running Java process.
 ```
-asprof -e cpu,alloc,lock -f profile.jfr ...
+$ asprof -d 30 -f /tmp/flamegraph.html <PID>
 ```
-or use `--alloc` and `--lock` parameters with the desired threshold:
-```
-asprof -e cpu --alloc 2m --lock 10ms -f profile.jfr ...
-```
-The same, when starting profiler as an agent:
-```
--agentpath:/path/to/libasyncProfiler.so=start,event=cpu,alloc=2m,lock=10ms,file=profile.jfr
-```
-
-## Flame Graph visualization
-
-async-profiler provides out-of-the-box [Flame Graph](https://github.com/BrendanGregg/FlameGraph) support.
-Specify `-o flamegraph` argument to dump profiling results as an interactive HTML Flame Graph.
-Also, Flame Graph output format will be chosen automatically if the target filename ends with `.html`.
-
-```
-$ jps
-9234 Jps
-8983 Computey
-$ asprof -d 30 -f /tmp/flamegraph.html 8983
-```
-
-[![Example](https://github.com/async-profiler/async-profiler/blob/master/demo/flamegraph.png)](https://htmlpreview.github.io/?https://github.com/async-profiler/async-profiler/blob/master/demo/flamegraph.html)
-
-## Profiler Options
-
-`asprof` command-line options.
-
-* `start` - starts profiling in semi-automatic mode, i.e. profiler will run
-  until `stop` command is explicitly called.
-
-* `resume` - starts or resumes earlier profiling session that has been stopped.
-  All the collected data remains valid. The profiling options are not preserved
-  between sessions, and should be specified again.
-
-* `stop` - stops profiling and prints the report.
 
-* `dump` - dump collected data without stopping profiling session.
+[![Sample FlameGraph](https://github.com/async-profiler/async-profiler/blob/master/demo/flamegraph.png)](https://htmlpreview.github.io/?https://github.com/async-profiler/async-profiler/blob/master/demo/flamegraph.html)
 
-* `check` - check if the specified profiling event is available.
+For the detailed walkthrough, please refer to the
+[Getting Started Guide](https://github.com/async-profiler/async-profiler/blob/master/docs/GettingStarted.md).
 
-* `status` - prints profiling status: whether profiler is active and
-  for how long.
+# Output formats
 
-* `meminfo` - prints used memory statistics.
+async-profiler can generate profile outputs in multiple formats which have been explained in details in the
+[Output Formats Documentation](https://github.com/async-profiler/async-profiler/blob/master/docs/OutputFormats.md)
 
-* `list` - show the list of profiling events available for the target process
-  (if PID is specified) or for the default JVM.
+# Profiling modes
```

Comment:
A link to Profiling Modes chapter can be added in the About section which lists profiling events:

> async-profiler can trace the following kinds of events:
> 
> - CPU cycles
> - Hardware and Software performance counters like cache misses, branch misses, page faults, context switches etc.
> - Allocations in Java Heap
> - Contented lock attempts, including both Java object monitors and ReentrantLocks
> - and more [link]

Reply:
Addressed in [adc296d](https://github.com/async-profiler/async-profiler/pull/1029/commits/adc296d22ffe7bcf227f89b9b92e799a2f72f2f0)